src/System.Runtime.Loader/ref/System.Runtime.Loader.csproj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="System.Reflection.Metadata.cs" />
```

Comment:
Generally we keep all the APIs in the single cs file that matches the name of the contract. @nguerrera were you thinking we should change this given you signed off?